app/src/configs/env.ts

```diff
@@ -35,6 +36,9 @@ export const ENABLE_PLANS = setEnv('REACT_APP_ENABLE_PLANS', 'false') === 'true'
 /** Activate the teams menu */
 export const ENABLE_TEAMS = setEnv('REACT_APP_ENABLE_TEAMS', 'false') === 'true';
 
+/** Activate the healthcare menu */
+export const ENABLE_HEALTHCARE_SERVICES = setEnv('REACT_APP_ENABLE_FHIR_HEALTHCARE_SERVICES', 'false') === 'true';
```

Comment:
nitpick: I feel like changes should be sequential (new changes added to end of file) or co-located with other similar changes